crates/miden-lib/asm/kernels/transaction/lib/epilogue.masm

```diff
@@ -280,10 +231,6 @@ export.finalize_transaction
     exec.update_account_code
```

Comment:
I would treat `update_account_code` in the same way as `update_account_storage_commitment` (even though it is just a placeholder for now).

Reply:
I moved `update_account_code` to the account and added a call to it in `compute_current_commitment`. Not completely sure if that's what you meant though.